rpm/mtools.spec

```diff
@@ -11,7 +11,7 @@ Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 Requires(post): /sbin/install-info
 Requires(postun): /sbin/install-info
```

Comment:
Drop these and also %post %preun?

Also drop Group: tags and %clean section.

Reply:
Ok, removed:
Group: Applications/System
Requires(post): /sbin/install-info
Requires(postun): /sbin/install-info
%clean
%post
%preun